src/test/resources/application-test.yml

```diff
@@ -57,7 +57,8 @@ spring:
         Accept: application/json
   flyway:
     out-of-order: true
-
+  jpa:
+    show-sql: true
```

Comment:
I don't think we should put this into `main`

Reply:
Yeah slipped in there, removed 👍